src/components/Template/Template.tsx

```diff
@@ -47,6 +50,16 @@ const Template = (props: React.PropsWithChildren<TemplateProps>) => {
   );
 };
 
+/**
+ * This optional component renders its children from edge-to-edge and should
```

Comment:
I didn’t see anything in the updated documentation that talked about the a11y reasoning for adding this template region. Can you add something?